src/cf-task-pool.js

```diff
@@ -29,18 +27,26 @@ class CFTaskPool {
   }
 
   canStartBuild(build) {
-    const requestedMemory = this._requiresCustom(build) ? this._taskCustomMemory : this._taskMemory;
+    const requestedMemory = this._containerSize(build).memory_in_mb;
     return this._hasAvailableMemory(requestedMemory);
   }
 
-  start() {
-    return this._setTaskAppGUID(this._taskAppName);
-  }
-
   async startBuild(build) {
-    const buildTask = this._buildTask(build);
+    const containers = await this._apiClient.fetchBuildContainersByLabel();
+    if (containers.length < 1) {
+      throw new TaskStartError('No build containers exist in this space.');
+    }
+
+    const containerName = build.containerName || 'default';
+    const container = containers.find(c => c.containerName === containerName);
+    if (!container) {
+      throw new TaskStartError(`Could not find build container with name: "${containerName}"`);
+    }
+
+    const buildTask = this._buildTask(build, container.command);
```

Comment:
Was wondering what you were thinking with the command annotation on the garden build apps. 🥽

Reply:
The experimental build container uses a different command to launch the build because it contains the self-contained executable instead of plain 'ol python code. Using the annotation allows this information to be specified alongside the container deployment (in the manifest) and the builder to not require knowledge of or configuration for the command. This frees us up to change this or have specific arguments/commands for specific build containers.

An "annotation" is required instead of a "label" because the "labels" have more restrictive character limitations since they can be used for searching/filtering in the API while the "annotation"s allow for any characters, such as spaces.